internal/cloudapi/v2/depsolve.go

```diff
@@ -86,7 +86,10 @@ func (request *DepsolveRequest) Depsolve(df *distrofactory.Factory, rr *reporegi
 			return nil, HTTPErrorWithInternal(ErrorFailedToDepsolve, err)
 		}
 
-		packageSet = manifestSource.GetPackageSetChains()
+		packageSet, err = manifestSource.GetPackageSetChains()
+		if err != nil {
+			return nil, HTTPErrorWithInternal(ErrorFailedToDepsolve, err)
```

Comment:
This is a bit of an open question, i.e. if we should create a new ErrorFailedToGetPkSetChains or if we lump it under the existing error, please advise, happy to tweak this.